src/utils/query/queryClient.ts

```diff
@@ -0,0 +1,3 @@
+import { QueryClient } from '@tanstack/react-query';
+
+export const queryClient = new QueryClient();
```

Comment:
I'm not sure if this is the best place for this... I was also considering a subdirectory in `src/hooks` :man_shrugging: